### PR TITLE
correct EL staking value

### PR DESCRIPTION
**Pull request by copyNdpaste**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/elysia/index.js exports TVL: 
                
 
 
bsc-staking               270.03 k
bsc                       806.08 k
ethereum-staking          526.28 k
ethereum                  3.12 M
staking                   796.31 k

total                    3.93 M

**Comment by copyNdpaste** [comment]
if divide `EL` staking value by 1e18 is `2.0515681178160253e-12`. I think it is result of twice dividen.
so i removed `/ 1e18`

image

image

File: projects/elysia/index.js [hunk 1]
```diff
@@ -4,63 +4,87 @@ const apiInfo = require("./apiInfo.json");
...
+    ),
+    axios.get(apiInfo["coin-gecko"].endpoint),
+  ]);
+
+  prices = coinGeckoCacheResponse.data;
+  prices.elfi = {};
+  prices.elfi.usd = Number(
+    uniswapV3SubgraphCacheResponse.data.data.daiPool.poolDayData[0].token1Price
+  );
+})();
```
Comment (waynebruce0x) [inline projects/elysia/index.js:37]: Can you remove this since its not used?

File: projects/elysia/apiInfo.json [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+    "endpoint": "https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3",
+    "body": {
+      "query": "{\n    daiPool: pool(id: \"0xbde484db131bd2ae80e44a57f865c1dfebb7e31f\"){\n    totalValueLockedUSD,\n    totalValueLockedToken0,\n    totalValueLockedToken1,\n    liquidity,\n    sqrtPrice,\n    poolDayData(orderBy: date, orderDirection: desc) {\n        date,\n        token1Price,\n    },\n    }\n    ethPool: pool(id: \"0xc311faebe8802f9cfc91284016d1de9537ec66b7\"){\n    totalValueLockedUSD,\n    totalValueLockedToken0,\n    totalValueLockedToken1,\n    liquidity,\n    sqrtPrice,\n    poolDayData(orderBy: date, orderDirection: desc) {\n        date,\n        token1Price,\n    },\n    }\n    stakedDaiPositions: positions(where: {pool: \"0xbde484db131bd2ae80e44a57f865c1dfebb7e31f\", owner: \"0x1f98407aaB862CdDeF78Ed252D6f557aA5b0f00d\"}){\n    id,\n    liquidity,\n    depositedToken0,\n    depositedToken1,\n    }\n    stakedEthPositions: positions(where: {pool: \"0xc311faebe8802f9cfc91284016d1de9537ec66b7\", owner: \"0x1f98407aaB862CdDeF78Ed252D6f557aA5b0f00d\"}){\n    id,\n    liquidity,\n    depositedToken0,\n    depositedToken1,\n    }\n  }"
+    }
+  },
```
Comment (waynebruce0x) [inline projects/elysia/apiInfo.json:8]: I dont think the edits to this file are required

**Comment by llamatester** [comment]
The adapter at projects/elysia/index.js exports TVL: 
        
 
 
    bsc-staking               289.64 k
    bsc                       805.10 k
    ethereum-staking          558.81 k
    ethereum                  3.07 M
    staking                   848.45 k
    
    total                    3.87 M

File: projects/elysia/apiInfo.json (hunk 2 above)
Comment (copyNdpaste) [reply projects/elysia/apiInfo.json]: @waynebruce0x I removed!

**Pull request merged**